django/thunderstore/frontend/api/experimental/tests/test_package_details.py

```diff
@@ -0,0 +1,192 @@
+from typing import Dict
+
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+from thunderstore.community.factories import PackageListingFactory
+from thunderstore.community.models import PackageListing
+from thunderstore.repository.factories import (
+    PackageRatingFactory,
+    PackageVersionFactory,
+)
+
+
```

Comment:
I'd expect tests for approval status visibility rules to be included

Reply:
Adding tests.